lib/assertions.js

```diff
@@ -1619,11 +1625,13 @@ module.exports = expect => {
                       } else {
                         return output.clone().block(function() {
                           if (type === 'moveSource') {
-                            this.property(
-                              diffItem.actualIndex,
-                              inspect(diffItem.value),
-                              true
-                            )
+                            subjectType
+                              .property(
+                                this,
```

Comment:
Could you get the output as an argument for the block instead, fiddling around with `this` becomes hard to follow. I regret I bound this for the magicpen API.